### DISCUSSION TITLE
Toast not showing in Storybook

### DISCUSSION THREAD

**Discussion by kalkusa** [chakra-ui/chakra-ui#7639] [Q&A]
Hi guys.

In the project I work in we use custom Chakra toasts. Those toasts are documented in Storybook. After recent upgrade they stopped working. They show properly in the app, but not in Storybook.

Storybook has few sections and the section where Chakra-UI components are listed works fine. At first I thought that it might be some issue with our project configuration. I created completely new test project to verify it. It seems that in plan new project also toasts don't show up in Storybook - custom or standard toast, nevermind. Yet still they show in Chakra-UI section. Just not project section.

Here I'm sharing the code: https://github.com/kalkusa/chakra-storybook-toast-test/tree/main/chakra-test

Does anyone know if something special needs to be done to handle Chakra toasts in Storybook?

**Comment by ramonmulia** [comment]
the only way I was able to make Toast to work in storybook was using "createStandaloneToast" function.
